make/release_flags.mk

```diff
@@ -11,7 +11,7 @@ windows-386 \
 windows-amd64 \
 windows-arm
 
-LND_RELEASE_TAGS = litd autopilotrpc signrpc walletrpc chainrpc invoicesrpc watchtowerrpc
+LND_RELEASE_TAGS = litd autopilotrpc signrpc walletrpc chainrpc invoicesrpc watchtowerrpc routerrpc wtclientrpc
```

Comment:
Ohhh... This is quite sneaky... When we set the `routerrpc` tag here for example, then the integrated `lnd` (and the one used in our itest!) will show the tag in the version tag list. But the real `lnd` won't. So we need to make sure this really reflects the current list of build tags in `lnd`, otherwise our itests might give us false positives.